db/warehouse/migrate/20241216184819_talent_configuration_flags.rb

```diff
@@ -0,0 +1,6 @@
+class TalentConfigurationFlags < ActiveRecord::Migration[7.0]
+  def change
+    add_column :configs, :default_lms_email_to_warehouse_email, :boolean
+    add_column :talentlms_configs, :allow_automatic_redirect_to_course, :boolean
```

Comment:
I think this should default to true so we don't break existing installations right?
```suggestion
    add_column :talentlms_configs, :allow_automatic_redirect_to_course, :boolean, default: true
```

Reply:
Yes, defaulting this to true will keep current installations more closely aligned with the current setup. I'll include it in the next commit.